pages/glossary/adjacent.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Adjacent
+key: adjacent
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+[Bounding boxes][] A and B are _adjacent_ if they do not [overlap][] and at least one point from box A is at the Euclidean distance of 1 [CSS pixel][] from one point in box B.
```

Comment:
I'm a little nervous about this being exactly 1px with no margin of error. I think we should include anything less than 2, and greater than 0.